src/email/templates/Verify/VerifyText.ts

```diff
@@ -1,11 +1,11 @@
-import { Routes } from '@/shared/model/Routes';
-
 export const VerifyText = () => `
+Thank you for $registerAction
+
 Welcome to the Guardian,
 
-Please click below to verify your emails address and complete your registration.
+Please click below to complete your registration.
 
-https://profile.theguardian.com${Routes.WELCOME}
+$verificationLink
```

Comment:
Again, happy to go either way, but the other emails construct the url by only inserting the token. Shall we move everything over to this method?